test/.project

```diff
@@ -4,6 +4,7 @@
 	<comment></comment>
 	<projects>
 		<project>iaf-core</project>
+		<project>ibis-adapterframework-ifsa</project>
```

Comment:
Dit moet je niet standaard in de `project` file opnemen. Het is een proprietary module! (en de naam klopt ook niet, dan zou het `iaf-ifsa` moeten zijn?)